lib/net.js

```diff
@@ -162,6 +162,16 @@ function isPipeName(s) {
   return typeof s === 'string' && toNumber(s) === false;
 }
 
+/**
+ * Creates a new TCP or IPC server
+ * @param {{
+ * allowHalfOpen?: boolean;
+ * pauseOnConnect?: boolean;
+ * }} options
+ * @param {Function} connectionListener
```

Comment:
```suggestion
/**
 * Creates a new TCP or IPC server
 * @param {{
 *   allowHalfOpen?: boolean;
 *   pauseOnConnect?: boolean;
 *   }} [options]
 * @param {Function} [connectionListener]
```
These 2 parameters are optional + formatting to keep consistency with the JSDocs in other lib modules.